website/data/docs-navigation.js

```diff
@@ -195,6 +195,7 @@ export default [
             content: ['compatibility'],
           },
           'uninstall',
+          'server-certificate-rotation',
```

Comment:
I'm thinking that maybe we should call it just "certificate rotation" to leave space for other cert rotation docs, e.g. CAs. wdyt?

Reply:
that makes sense. I wanted to be explicit for now hoping that in the future, when we add more details to this page regarding cert rotation for other certificates (CA certs), we would update it, but now seems as good as later